### PR TITLE
obj.get_reaches and other mods

### PR DESCRIPTION
**Pull request by anatoliy-savchak**
* obj.get_reaches() to return tuple of (reach and min reach), as per #644;
* obj.anim_goal_push_walk_to_tile() to make critter walk to location;
* obj.anim_goal_push_run_to_tile() to make critter run to location;
* LocXY expanded usage;
* LocAndOffsets expanded usage;
* LocFull expanded usage;

### CODE REVIEW

File: TemplePlus/location.h [hunk 1]
```diff
@@ -3,6 +3,9 @@
 #include "common.h"
 #include <temple/dll.h>
 
+const float PIXEL_PER_TILE = sqrt(800.0f);
```
Comment (DudeMcDude) [inline TemplePlus/location.h:6]: Revert this please

Comment (anatoliy-savchak) [reply TemplePlus/location.h]: done

**Comment by anatoliy-savchak** [comment]
@DudeMcDude, it' ready for approve and merge. Please consider it.

**Comment by anatoliy-savchak** [comment]
Outdated, closing...

**Pull request closed without merging**